src/components/comment-media/comment-media.module.css

```
@@ -169,6 +169,69 @@
+  max-width: 30ch;
+  font-size: 8pt;
+}
+
+.mediaLoadFailureStatus {
+  position: absolute;
+  width: 1px;
+  height: 1px;
+  padding: 0;
+  margin: -1px;
+  overflow: hidden;
+  clip: rect(0 0 0 0);
```

Stylelint (17.11.0)

[error] Deprecated property "clip" (property-no-deprecated)

(property-no-deprecated)